taskfile.js

```diff
@@ -0,0 +1,36 @@
+export async function compile(task) {
+  await task.parallel(['src']);
+}
+
+export async function src(task, opts) {
+  await task
+    .source(
+      opts.src || [
+        'src/**/*.+(js|jsx|ts|tsx)',
+        '!src/**/*.spec.+(js|jsx|ts|tsx)',
+        '!src/**/*.snap',
+      ],
+    )
+    .babel({
```

Comment:
Why is `.babel` needed here? 

Does the root `babel.config.js` not get inherited?

Reply:
🔥️ removed `taskr` went back to `@babel/cli`